### PR TITLE
Update to GHC 8.8

### PR DESCRIPTION
**Pull request by Avi-D-coder**

### CODE REVIEW

**Comment by Avi-D-coder** [comment]
I'll regenerate Travis once `haskell-ci` finishes building

File: stack.yaml.lock [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
```
Comment (mpickering) [inline stack.yaml.lock]: Don't think you should commit this file?

Comment (Avi-D-coder) [reply stack.yaml.lock]: I'm a bit confused. That's the whole file. Should I remove it and or add it to `gitignore`?

Comment (mpickering) [reply stack.yaml.lock]: I don't have any idea about what this file does but @jneira advided today that we should not commit them.

Comment (Avi-D-coder) [reply stack.yaml.lock]: Sounds good, I'll add it to `gitignore`

**Comment by bubba** [comment]
Will there be a Hackage release for this?

**Pull request merged**